AUTHORS

```diff
@@ -10,6 +10,7 @@ Andreas Schildbach
 Bennett Hoffman
 Carlos Lopez-Camey
 Carsten Otto
+Chris Pacia
```

Comment:
Note that on the next release, this will be auto-generated from your author info in the commit. Feel free to keep this chunk though.